adobe/spectrum-web-components
# Update Spectrum CSS version

## Description
- adopts the updated color pallet from Spectrum CSS https://spectrum.adobe.com/page/whats-new/
- updates the implementation of `<sp-menu-divider>` to leverage the `<sp-divider>` styles
- ~need to do one confirmation with Spectrum CSS of whether this should REMOVE `lightest` or merge it's values with `light` before moving forward.~

## Motivation and context
Keeping up with Spectrum

## How has this been tested?
    1. Go [here](https://73874bdaecee6ea6d7a506b37d5d256d--spectrum-web-components.netlify.app/review/#MenuGroupStories/inherit.png)
    2. See the divider lines are darkened
    3. Go [here](https://spectrum-css--spectrum-web-components.netlify.app/storybook/?path=/story/menu-group--mixed)
    4. See that the darkness is persisted
    1. Go [here](https://73874bdaecee6ea6d7a506b37d5d256d--spectrum-web-components.netlify.app/review)
    2. See that the colors are updated across ALL of the system 🤯 

## Types of changes

## Checklist
